web/src/utils/formatting.ts

```diff
@@ -11,18 +11,29 @@ function addSpaceBetweenNumberAndUnit(inputString: string) {
 
 export const formatEnergy = function (
   d: number,
-  numberDigits: number = DEFAULT_NUM_DIGITS
+  numberDigits: number = DEFAULT_NUM_DIGITS,
+  isCapacity = false
 ) {
   // Assume MW input
   if (d == undefined || Number.isNaN(d)) {
     return d;
   }
   const significantFigures = d.toString().length > 1 ? numberDigits : 1;
-
-  const power = `${d3.format(`.${significantFigures}s`)(d * 1e6)}Wh`;
+  const unit = isCapacity ? 'W' : 'Wh';
```

Comment:
Since we are doing this we might as well do it for the hourly production values as well so they match the capacity values?

Reply:
I think the idea is that capacity is an instantaneous measure and our hourly values are hourly so the mismatch is intentional. It could be confusing that when talking production values we provide different base units depending on time aggregation

Reply:
Agreed but MW and MWh has the same value since we are showing things in hourly resolution anyhow.

Reply:
Seems reasonable 👍